src/modules/apis.js

```diff
@@ -0,0 +1,99 @@
+import { MOVIE_API_URL, BASE_URL, movieContainer } from './variables.js';
+
+/* eslint no-underscore-dangle: ["error", {"allow": ["_embedded"]}] */
```

Comment:
Kindly do not disable the linters since they serve as valuable tools to maintain code quality, catch potential bugs, enforce best practices, and improve maintainability. I believe you can work around this one without disabling it. Therefore, kindly consider removing this line and ensure to work within the confines of the linters. :pray:

Reply:
We have resolved it. Thank you